hack/tests/subcommand.sh

```diff
@@ -16,32 +16,37 @@ operator-sdk test local ./test/e2e --global-manifest deploy/crds/cache.example.c
 
 # we use the test-memcached namespace for all future tests, so we only need to set this trap once
 kubectl create namespace test-memcached
-trap_add 'kubectl delete namespace test-memcached || true' EXIT
-operator-sdk test local ./test/e2e --namespace=test-memcached
+trap_add 'kubectl delete --ignore-not-found namespace test-memcached' EXIT
+operator-sdk test local ./test/e2e --deploy-namespace=test-memcached
 kubectl delete namespace test-memcached
 
 # test operator in up local mode
 kubectl create namespace test-memcached
-operator-sdk test local ./test/e2e --up-local --namespace=test-memcached
+operator-sdk test local ./test/e2e --up-local --deploy-namespace=test-memcached
+kubectl delete namespace test-memcached
+
+# test operator in up local mode with --watch-namespace flag
+kubectl create namespace test-memcached
+operator-sdk test local ./test/e2e --up-local --deploy-namespace=test-memcached --watch-namespace=""
```

Comment:
It still not clear for me. What actually will occurs with `--deploy-namespace=test-Memcached --watch-namespace=""`? Is not it the same of  use only ` --watch-namespace=""`?